src/optimizer/join_order_optimizer.cpp

```diff
@@ -257,20 +250,18 @@ static void UpdateExclusionSet(JoinRelationSet *node, unordered_set<idx_t> &excl
 	}
 }
 
-void printRelation2tableNameMapping(unordered_map<idx_t, std::string> relation_to_table_name) {
-	std::string res = "";
-	unordered_map<idx_t, std::string>::iterator it;
-	for (it = relation_to_table_name.begin(); it != relation_to_table_name.end(); it++) {
-		res += "{ " + std::to_string(it->first) + ": " + it->second + "}," + "\n";
-	}
-	res += "-----------\n";
-	std::cout << res << std::endl;
-}
+//void printRelation2tableNameMapping(unordered_map<idx_t, std::string> relation_to_table_name) {
```

Comment:
For the PR we'll need to clean some of this debugging code up. We don't want commented and/or unused code.

I imagine that some of it is no longer needed, but some of it will be nice to keep. If you look through the code base, you'll see that classes such as `LogicalOperator`/`DataChunk`/`Pipeline` have `ToString()` and `Print()` methods.

Could you check if some of the debugging code can be removed and/or refactored to `ToString()` and `Print()` methods?